### PR TITLE
Fixed picked up item's inventory slot

### PR DESCRIPTION
**Pull request by Czompi**

### CODE REVIEW

File: Obsidian.API/Containers/Container.cs [hunk 1]
```diff
@@ -30,27 +30,12 @@ public override int AddItem(ItemStack item)
...
-                    invItem.Count += item.Count;
-
-                    return i;
-                }
-
-                if (invItem == null)
-                {
-                    this.items[i] = item;
+            int player_hotbar_start = 36, player_hotbar_end = player_hotbar_start + 8;
+            int player_main_inventory_start = 9, player_main_inventory_end = player_hotbar_start - 1;
```
Comment (Naamloos) [inline Obsidian.API/Containers/Container.cs]: Generally constant values are defined in the beginning of a class, in all caps. e.g. const int PLAYER_HOTBAR_START, PLAYER_HOTBAR_END

**Review by Naamloos — CHANGES_REQUESTED** [review]

File: Obsidian.API/Containers/Container.cs (hunk 1 above)
Comment (Naamloos) [reply Obsidian.API/Containers/Container.cs]: that aside looks great

Comment (Seb-stian) [reply Obsidian.API/Containers/Container.cs]: In C#, constants are **always** in PascalCase.

Comment (Naamloos) [reply Obsidian.API/Containers/Container.cs]: oops. yet still they don't belong inline

File: Obsidian.API/Containers/Container.cs [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 public sealed class Container : BaseContainer, IBlockEntity
 {
+    private const int PLAYER_HOTBAR_START = 36;
+    private const int PLAYER_HOTBAR_END = PLAYER_HOTBAR_START + 8;
+    private const int PLAYER_MAIN_INVENTORY_START = 9;
+    private const int PLAYER_MAIN_INVENTORY_END = PLAYER_HOTBAR_START - 1;
+
```
Comment (Czompi) [inline Obsidian.API/Containers/Container.cs]: I fixed the naming here

File: Obsidian.API/Containers/Container.cs (hunk 1 above)
Comment (Czompi) [reply Obsidian.API/Containers/Container.cs]: > In C#, constants are **always** in PascalCase.

I'll change it.

Comment (Czompi) [reply Obsidian.API/Containers/Container.cs]: Changed it.

**Review by Naamloos — APPROVED** [review]

**Pull request merged**